contracts/src/TokenSpoke/interfaces/INativeTokenSpoke.sol

```diff
@@ -19,7 +19,12 @@ import {ITokenSpoke} from "./ITokenSpoke.sol";
  */
 interface INativeTokenSpoke is ITokenSpoke, INativeTokenBridge {
     /**
-     * @dev Emitted when reporting burned tx fees to hub chain.
+     * @notice Emitted when native tokens are burned for bridge transfers.
+     */
+    event BurnedForBridge(uint256 amount);
```

Comment:
Would love to get thoughts on naming of event, and if this should only apply to `INativeTokenSpoke`

Reply:
Ah sorry didn't see this comment in time. But left a comment suggesting we remove it all together.